parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetWriter.java

```diff
@@ -537,6 +539,21 @@ public SELF withPageWriteChecksumEnabled(boolean enablePageWriteChecksum) {
       return self();
     }
 
+    /**
+     * Enables bloom filter column names for the constructed writer.
+     *
+     * @return this builder for method chaining.
+     */
+    public SELF withBloomFilterColumnNames(String columnNames) {
```

Comment:
`ParquetWriter.Builder` is to be used from code. I think, it would be more usable if we would expect varargs of strings or we would have a different method that can add column names one-by-one.